build/Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+pipeline {
+  agent {
+      label "gcp-agent"
+  }
+  stages {
+    stage('setup') {
+      steps {
+          sh '''
+          ./tf-wrapper.sh setup $BRANCH_NAME
```

Comment:
need to align with the impersonate step as discussed offline

Reply:
Jenkinsfile now has the extra step and it is using the same command as the `cloud-build-tf-plan.yaml` and `cloud-build-tf-apply.yaml` files:

```
    echo "Setting up gcloud for impersonation"
    gcloud config set auth/impersonate_service_account ${_TF_SA_EMAIL}
    echo "Adding bucket information to backends"
    for i in `find -name 'backend.tf'`; do sed -i 's/UPDATE_ME/${_STATE_BUCKET_NAME}/' $i; done
```